Report/QuantConnect.Report.csproj

```diff
@@ -179,6 +179,9 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
     <None Include="packages.config" />
+    <None Include="Python.Runtime.dll.config">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
I think if possible it would be better to move this `.config` file into the shared `QuantConnect` common project, so we cover these different cases in a single place/file. With the benefit of also having to request users to modify it only once in a single place